content/2017/02/17/color-theory-for-coders.rst

```diff
@@ -0,0 +1,37 @@
+public: yes
+author: sondra
+tags: [Tips, Color, Design, Recommendations]
+image:
+  - src: '2017/color-theory/decision-tree.jpg'
+summary: |
+  If you write code and you want to delve into design, Natalya Shelburne's Practical Color Theory for Coders will get you up and running with a beautiful, cohesive, and accessible color palette using Sass color functions and teach you why her palettes work so well all at the same time.
+
+
+Birds Recommend: Color Theory for Coders
+=======================================
+
+So you write code, but design intrigues you and you'd like to delve deeper. But where to start? The sheer number of tools and tutorial is overwhelming. My article on the best design tools and tutorials will go live next week, but while you're waiting, checkout `Natalya Shelburne's`_ `Practical Color Theory for People Who Code`_. Continue reading for a quick summary or **head
+over to Natalya's article right away!**
+
+.. _Natalya Shelburne's: https://twitter.com/natalyathree
+.. _Practical Color Theory for People Who Code: http://tallys.github.io/color-theory/
+
+`Go to Color Theory >`_
+-----------------------
+
+.. _Color Theory >: http://tallys.github.io/color-theory/
+
+In Practical Color Theory, Natalya—fine artist, speaker, and front end developer—demos the Sass color functions she created. Start with any hsl color you like, maybe your client's brand color. Color functions take all the guess work out of picking a second color. Mix a bit of each color into the other to establish a relationship. Add neutral colors so you don't overwhelm the eye. Add contrast for accessibility and keep your color palette limited for a reasonablity level of complexity. Voila! You're designing with color.
```

Comment:
- Style guide also suggests en-dash with space around it, instead of em-dash un-spaced, for asides like this one. 
- Could link `speaker` back to our CSS Day recap, where she gave the talk?
- (line length again)
- Some of these sentence structures are hard to follow. I don't have specific thoughts on that off-hand. Review for clarity?